lib/attack.go

```diff
@@ -328,6 +328,10 @@ func (a *Attacker) hit(tr Targeter, name string) *Result {
 	res.Latency = time.Since(res.Timestamp)
 	res.BytesIn = uint64(len(res.Body))
 
+	if r.ContentLength != -1 {
+		res.BytesIn = uint64(r.ContentLength)
```

Comment:
Hmm, this doesn't look right. `BytesIn` is supposed to say how many actual bytes were read from the response, not how many bytes could be read.

Reply:
To me bytes in should be the size of the body regardless of whether you read it all in or not.  I would like to get the true size of the body even if I limit it with `MaxBody`.

Another option would be to use content length only if `MaxBody` is set.  An additional option would be to add a new field to represent this number and leave bytes in as is.

Let me know what your thoughts are!

Reply:
> Another option would be to use content length only if MaxBody is set. An additional option would be to add a new field to represent this number and leave bytes in as is.

This sounds like the correct option to me.